account_invoice_line_sequence/views/report_invoice.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
```

Comment:
`<openerp>` can be changed to `<odoo>`
`<data>` tag is dispensable when `noupdate="0"`